README.md

```diff
@@ -347,28 +352,38 @@ export default Ember.Route.extend({
 });
 ```
 
-## Disabling incompatible dependencies
+## Build Hooks for FastBoot
+
+### Disabling incompatible dependencies
 
 There are two places where the inclusion of incompatible JavaScript libraries could
 occur:
 
  1. `app.import` in the application's `ember-cli-build.js`
  2. `app.import` in an addon's `included` hook
 
-`ember-cli-fastboot` sets the `EMBER_CLI_FASTBOOT` environment variable when it is
-building the FastBoot version of the application. You can use this to prevent the
-inclusion of the library at build time:
+You can include the incompatible Javascript libraries by wrapping them with a `FastBoot` variable check. In the browser, `FastBoot` global variable is not defined.
 
 ```js
-if (!process.env.EMBER_CLI_FASTBOOT) {
-  // This will only be included in the browser build
-  app.import('some/jquery.plugin.js')
+var map = require('broccoli-stew').map;
+
+treeForVendor(defaultTree) {
+  var browserVendorLib = new Funnel(...);
+
+  browserVendorLib = map(browserVendorLib, (content) => `if (typeof FastBoot === 'undefined') { ${content} }`);
```

Comment:
should we tell people to just send the files to a different outputFile and have that included as part of the fastboot manifest? or.. ?